package.json

```diff
@@ -88,13 +88,14 @@
     "command-line-usage": "^5.0.5",
     "epipebomb": "^1.0.0",
     "fs-extra": "^7.0.1",
-    "highlight.js": "github:marcoscaceres/highlight.js#7b144b3b83ce296d5b29c5e4fc2ea2a65e9d3f32",
+    "highlight.js": "git+ssh://git@github.com/highlightjs/highlight.js.git#cli",
     "hyperhtml": "^2.25.4",
     "loading-indicator": "^2.0.0",
     "marked": "^0.6.0",
+    "npm-check-updates": "^2.15.0",
```

Comment:
Is this intentional?

Reply:
oh no, definitely not.

Reply:
stupid `ncu` keeps asking to update, but locally... even though I have it installed globally, so keep screwing that up.